src/pages/policy/output/budget/MultiYearBudgetaryImpact.jsx

```diff
@@ -3,15 +3,15 @@ import { Table } from "antd";
 const usDataSources = [
   {
     header: "Federal tax",
-    budgetKey: "budgetary_impact",
+    budgetKey: "tax_revenue_impact",
```

Comment:
This is still inconsistent I believe: federal tax revenue impact is `tax_revenue_impact 
- state_tax_revenue_impact`.

I would look at [this](https://github.com/PolicyEngine/policyengine-app/blob/02049539511509ad251839d2ca7e8ba0612cb15e/src/pages/policy/output/budget/BudgetaryImpact.jsx#L159) part of the codebase to see exactly which variables map to which concepts and just mirror that. Apologies for missing this in earlier reviews if this was part of one of the chained PRs: I think if in future we could just combine PR dependent on each other in the same repo that would make it easier to review.